### DISCUSSION TITLE
Field generated by alter migration not appears on request

### DISCUSSION THREAD

**Discussion by ibrahim-brumate** [prisma/prisma#6467] [Q&A] [answered]
Hi Guys.

I made one API with Fastify with docker to test prisma. When I made the first migration and use the findFirst everything works fine.

```ts
findFirst = async (queryConditions) => {
  return await prisma.user.findFirst({
    where: queryConditions,
    include: {
      profile: {
        select: {
          profile: true
        }
      }
    }
  })
}

findFirst({ user: 'user1'})

// returns 
{
  "data": {
    "user": {
      "id": 1,
      "profileId": 1,
      "name": "user1",
      "profile": {
        "profile": "Test"
      }
    }
  }
}
```

But when I changed the schema and made another migration to add one field, this field was created correctly on MySQL but not returns
on query. I have a new field on data base named Phone but the query still returning the same data above.

What am I doing wrong? Am I forgot to do something? It's seems like the query made by prisma not changed even I using `npx prisma generate`

Thanks for all support.

**Comment by ibrahim-brumate** [comment, marked as answer]
I found the issue. I need to run `npx prisma generate` on docker container and I was running on my local machine. I put this on compose.yml and solved the case. Thanks guys

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
